src/controllers/movies/movies.js

```diff
@@ -228,24 +235,24 @@ define(['loading', 'layoutManager', 'userSettings', 'events', 'libraryBrowser',
                     });
                 });
             }
-            var btnSelectView = tabContent.querySelector('.btnSelectView');
+            const btnSelectView = tabContent.querySelector('.btnSelectView');
             btnSelectView.addEventListener('click', function (e) {
                 libraryBrowser.showLayoutMenu(e.target, self.getCurrentViewStyle(), 'Banner,List,Poster,PosterCard,Thumb,ThumbCard'.split(','));
             });
             btnSelectView.addEventListener('layoutchange', function (e) {
-                var viewStyle = e.detail.viewStyle;
+                let viewStyle = e.detail.viewStyle;
                 userSettings.set(savedViewKey, viewStyle);
                 query.StartIndex = 0;
                 onViewStyleChange();
                 itemsContainer.refreshItems();
             });
         }
 
-        var self = this;
-        var itemsContainer = tabContent.querySelector('.itemsContainer');
-        var savedQueryKey = params.topParentId + '-' + options.mode;
-        var savedViewKey = savedQueryKey + '-view';
-        var query = {
+        let self = this;
```

Comment:
Seems like we can remove this line, unless I'm missing something.

Also in general, if we really need to alias `this`, it's better to use `that`, as `self` already points to `window` in this case, and it can be confusing.

Reply:
Thanks for noticing it. 
Unless I'm doing something wrong, when  you remove `let self = this` in both moviesrecommended and movies you get:
`moviesrecommended.js:368 Uncaught (in promise) TypeError: controller.renderTab is not a function`.
when accessing movies in the library.

Reply:
I have removed stopped ailiasing self in movies.js, would you mind checking it, and if it's all ok, I will do it on the rest of them? thanks